client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/allocation/allocateQuantities.test.ts

```diff
@@ -0,0 +1,186 @@
+import { FnUtils } from '@openmsupply-client/common';
+import { allocateQuantities } from './allocateQuantities';
+import { DraftStockOutLineFragment } from '../../../api/operations.generated';
+
+describe('allocateQuantities - standard behaviour.', () => {
```

Comment:
test cases are the same as old tests (removed no-longer-relevant cases like placeholder though)